.circleci/config.yml

```diff
@@ -78,6 +83,39 @@ jobs:
       - run: { name: 'Release S3', command: "cd patternlab/styleguide && node_modules/.bin/gulp patternlab:release" }
       - run: { name: 'Release NPM', command: "cd patternlab/styleguide && node_modules/.bin/gulp npm:release" }
 
+  react_build_storybook:
+    <<: *react_defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - mfr-dependencies-{{ checksum "react/package.json" }}
+      - run: cd react && npm install
+      - save_cache:
+          paths:
+            - react/node_modules
+          key: mfr-dependencies-{{ checksum "react/package.json" }}
+      - run:
+          name: Mayflower React Build
+          command: cd react && npm run build
+      - run:
+          name: Mayflower React Build Storybook
+          command: cd react && npm run build-storybook
+      - persist_to_workspace:
+          root: ~/repo
+          paths: .
+
+  react_deploy_tag:
+    <<: *react_defaults
+    steps:
+      - attach_workspace:
+          - at: ~/repo
+      - *configure_npm
+      - run: {name: 'Publish NPM package', command: 'cd react && npm publish'}
+      - run: {name: 'Sync storybook-static to S3', command: 'cd react && aws s3 sync storybook-static s3://mayflower-react.digital.mass.gov/ --delete'}
+      - run: cd react && aws configure set preview.cloudfront true
```

Comment:
is `cd react` necessary?

Reply:
No, but I thought it might be good to have all the commands consistent to (hopefully?) cut down on copy/paste errors in the future since almost every other command needs it.